feat(dashboard): gh #602 Added cluster banner + updated look and feel for delete button

abujeda opened the pull request:
Added cluster banner to the home page when no sessions are running.
Updated delete session button to make it less prominent.

Part of the UI improvements after the first deployment to the FASSE cluster.

abujeda committed:
feat(dashboard): gh #598 Added custom launchers and quick links per HPC cluster

whorka committed:
Update README.md

the Sid Dashboard is now expected to be working

whorka reviewed (changes requested):
This implementation hard-codes configuration values which are site-specific. Please retool the logic to load the `sid_welcome_message` HTML from a configuration file instead (to be delivered via the `docker compose` step in the `Makefile` in the Local Development and Remote Development environments; and to be delivered by Puppet in the Staging and Production environments).

In general, we should avoid hard-coding site-specific values, since this project is intended to be open source, and will be publicized to the wider OOD user community on Thursday.

whorka commented on dashboard/application/app/assets/stylesheets/quick_launch.scss:
```diff
@@ -7,6 +7,10 @@ $launcher-border-color: #337ab7;
 $launcher-border-color-secondary: #808080;
 $launcher-color-brown: #DA6027;
 
+$development-color: #255A8E;
+$cannon-color: #009CD2;
+$fasse-color: #DA6027;
```
site-specific

whorka committed:
Update README.md

Note that launching apps in the default dashboard is currently broken in Local Development environments.

abujeda committed:
fix(build): gh #598 fixed OOD dashboard session cookie creation for HTTP

abujeda commented:
@whorka started working on a different approach.
Making the cluster banner a widget and including the widget based on the current cluster id.

I still need to make more changes to complete this approach, but I would like to discuss it with @whorka

abujeda committed:
feat(dashboard): gh #602 Added cluster banner + updated look and feel for delete button

abujeda commented:
I created an improved approach based on the custom configuration per cluster changes from issue https://github.com/hmdc/DevOpsProjects/issues/598

This approach extends the quick links configuration file created for 598 to configure as well the welcome message.
Edit: Forced update from the improved branch as it is a better solution. To discuss at the daily.

abujeda committed:
feat(dashboard): gh #598 Added support for launcher color configuration to match cluster

abujeda commented:
Closing, changes added to #35
